ccmbioinfo/lfs
# LFS-1145: Modify set_admin_password.py so that it can also change passwords for non-admin accounts

This PR modifies `set_admin_password.py` and gives it the ability to set the password for non-admin Sling users.

To test:
1. Build this branch (`mvn clean install`)
2. Start CARDS
3. Create a new CARDS user named `test` with a password of `12345678`
4. `python3 Utilities/Administration/set_admin_password.py --user test`
5. Enter `test1234` for the `test` user password
6. `python3 Utilities/Administration/set_admin_password.py`
7. Enter `admin5678` for the `admin` user password
8. You should now be able to log in to CARDS as the `test` user with the password `test1234` and as the `admin` user with the password `admin5678`
